### DISCUSSION TITLE
how to set created by for registered models?

### DISCUSSION THREAD

**Discussion by tbolind** [mlflow/mlflow#8758] [Q&A]
Does anyone know how you set the created by keyword through the api?
Specifically I am talking about the created by seen in this picture:
![image](https://github.com/mlflow/mlflow/assets/136355482/e4e1017e-d218-4a4e-9c7b-4a24f1f981b1)
I would like these to be set automatically when a user runs and registers a model, so ideally I should be able to set them in python upon creation of the model.

**Comment by Nephdz** [comment]
@tbolind Were you able to figure this out? I have this same question
